### DISCUSSION TITLE
Mehrere parallele Rohre in einem Modell abbilden/Fußbodenheizung

### DISCUSSION THREAD

**Discussion by Labauke** [ghorwin/SIM-VICUS#146] [Ideas]
Anforderung:
Für einen Erdwärmekollektor/Fußbodenheizung wäre es schön in einem einzigen Modell mehrere identische parallele Rohre abzubilden, die alle mit der gleichen Außentemperatur Wärme austauschen.
Das ist eigentlich simpel und wäre fast identisch mit dem jetzigen DynamicPipe Model.

Einzige Änderung: zusätzlicher Parameter N_parallel
-> Bei der Berechnung der Geschwindigkeit aus dem Massenstrom würde man noch durch N_parallel teilen.
-> Für abgegebenen Gesamtwärmestrom würde man noch mit N_parallel multiplizieren

**Comment by ghorwin** [comment]
Hat Ähnlichkeiten mit dem Fußbodenheizungsmodell. Es hängt davon ab, ob die parallelen Rohre thermisch komplett unabhängig agieren, oder ob es wie bei Flächenheizungen sowas wie eine "Schichtwirkung" gibt. Bei typischen Erdwärmekollektoren würde ich das mal unterstellen. Damit wäre der oben skizzierte Ansatz wahrscheinlich falsch, da ja bei jedem Rohr eine einheitliche Umgebungstemperatur vorausgesetzt wird.

Grundsätzlich sehe ich ein Erdkollektor als komplexeres Modell, welches je nach Detailgrad durch ein entsprechendes Ersatzmodell abgebildet wird. Das ganz einfache Modell, was einfach konstante Temperaturen rund um das Rohr voraussetzt, ist wahrscheinlich zu falsch für eine sinnvolle Anwendung. Bei Einzelrohren mag das gehen, aber bei Kollektoren ist einfach Systembedingt die Energieentnahme zur "aktivierten" Fläche/Volumen zu groß. Daher würde ich so ein super simples Modell auch gar nicht erst einbauen (außer vielleicht testweise, um den Effekt zu klären, aber dann nur hard-gecoded in #if #endif Blöcken und ohne irgendwelche Datenstrukturen).

**Comment by Labauke** [comment]
Was genau meinst du mit Schichtwirkung?
Jedes Rohr hat die gleiche Umgebungstemperatur, da sie in der gleichen Tiefe verlegt sind, so werden bisher alle Erdwärmekollektoren gerechnet, anders ist es gar nicht praktikabel. Tatsächlich wird im Moment in der Praxis noch nichtmal mit einem Rohr gerechnet sondern nur noch volumetrisch Wärme entzogen, aber anderes Thema...
Bei einem mehrschichtigen Kollektor wird halt jede Schicht durch ein solches Modell abgebildet.

Ein großer Erdwärmekollektor hat >100 parallele Rohre, alle einzeln zu modellieren ist also nicht möglich. Und der Fehler ist nicht so groß mit dem einfachen Modell. 
Der Wärmeaustausch mit dem Erdreich findet ja nur in einer ausgewählten Schicht mit der Dicke der Rohrleitungen (25 mm) statt. Der Fehler hängt natürlich vom Verlegeabstand ab, können wir nochmal diskutieren.

In jedem Fall brauchen wir das Modell, eine andere Möglichkeit gibt es aus meiner Sicht nicht.

**Reply by KnutKnudson** [reply]
Sagt mal, das riecht doch nach einem Dimensionierungsproblem. Ist es denn nicht möglich, die vielen Rohre durch ein Einziges äquvalent abzubilden? 
Schaut auch mal den Testfall 'ThermoHydraulicNetworkParallelPipes.nandrad' an. Dort habe ich zweit Rohre so gestaltet, dass in dem zweiten exakt der doppelte Massestrom fließt und ein doppelter Wärmeverlust an die Umgebung abgegeben wird wie im ersten. Durch passende Parametrsiierung kann einziges Rohr kann folglich als Äquivalent für mehrere identische dienen.

**Reply by Labauke** [reply]
> Sagt mal, das riecht doch nach einem Dimensionierungsproblem. Ist es denn nicht möglich, die vielen Rohre durch ein Einziges äquvalent abzubilden?

Ja genau das ist mein Vorschlag, es ist eigentlich relativ simpel, siehe oben

**Comment by ghorwin** [comment]
Das Problem sind, glaub ich, nicht die Rohre sondern die Annahmen der Temperaturverteilung *um* die Rohre. Wenn man davon ausgehen kann, dass alle Rohre unabhängig voneinander die gleiche Umgebungstemperatur haben, dann ist es wirklich einfach. Aber bei einem Erdkollektor ist diese Annahme eben falsch - die Rohre beinflussen sich Gegenseitung und es gibt eine "Systemwirkung". 

Wer sich das mal vor Augen führen will, kann man ein einfaches Modell in DELPHIN aufsetzen und schauen, wie weit die Rohre auseinander liegen müssten, um einigermaßen gleichmäßige Umgebungstemperaturen zu haben.

Bei sehr vielen Rohren kann man wahrscheinlich ein äquivalentes Erdreichmodell bauen, d.h. man setzt an, dass der Wärmestrom von n  äquivalenten Einzelrohren dem Wärmestrom des Kollektorsystems ins Erdreich entspricht. In diese Umrechnung gehen dann:

- Verlegeabstand
- Rohreigenschaften (Geoemtrie, Dämmung)
- Erdreicheigenschaften (Wärmeleitung)
- Abstand zur Erdreichoberkannte

ein. Kann man gut mit DELPHIN durchspielen.

**Comment by Labauke** [comment]
Bei der Methode mit der wir bisher eigentlich fast ausschließlich Erdwärmekollektoren modelliert haben, treffen wir genau die Annahme:
Alle Rohre haben die gleiche Umgebungstemperatur.  

Dafür müssen sie keinen großen Abstand haben. Die Annahme besteht darin, dass jedes Rohr sozusagen unendlich viele Nachbarn hat, also das Kollektorfeld unendlich breit ist. Die Breite meint dabei die Ausdehnung senkrecht zur Fließrichtung. 
Damit kann man das Modell auf ein symmetrisches Modell reduzieren, welches nur noch ein halbes Rohr + den halben Rohrabstand beinhaltet. Die Annahme wird natürlich schlechter je kleiner der Kollektor ist.

**Comment by ghorwin** [comment]
Zusammenfassung Diskussion:

1. Fallunterscheidung: Fußbodenheizung und Erdkollektor

   + Fußbodenheizung: Annahme uniforme Umgebungstemperatur gerechtfertig (Schichtenaufbau, gut leitfähiger Estrich, darüber/darunter Dämmung, kleine Verlegeabstände --> Artikel zur Vergleichssimulation mit DELPHIN -> Dirk)
   + Erdkollektor: deutlich ausgeprägte 2D Temperaturverteilung, modelliert in symmetrie-Modellen DELPHIN -> Anbindung via FMU, für Spezialfälle (Parametergrenzen?) eventuell auch als "Fußbodenheizung" modellierbar (Prüfen --> Hauke)

2. Modellierung Fußbodenheizung:

Preprozessing: 

- Anwender gibt beheizte Fläche an (m2), und Verlegeabstand (cm)  -> gibt benötigte Rohrlänge
- integriert (Modellparameter?), maximale Einzelrohrlänge (m) -> ergibt Anzahl Felder/parallele Rohre/Rohrschlangen
- Unterteilung der Felder manuell durch Modellierer -> Hat Auswirkungen auf Wärmeübergabe

Aktuelle Annahme: jedes individuelle Feld (Fußbodenheizungsmodell) bekommt Umgebungstemperatur von exakt einer Konstruktionsschicht!

NANDRAD:

- Modellierung wie einzel Rohr; entweder SimplePipe oder DynamicPipe
- Zusätzliche Parameter: Skalierungsfaktor/Anzahl der parallelen Verlegung (n_pipes)
- HeatExchange-Parameter: Modell für Wärmeaustausch mit Konstruktionsschicht: identifiziert über ID (ConstruktionsInstance)

- *Konstruktionsinstanz* erhält neuen (optionalen) Parameter: Index der beheizten Schicht; Konstruktionsstatemodell veröffentlicht (falls angegeben) die mittlere beheizte Schichttemperatur (`ActiveLayerTemperature`) und das Konstruktionsbilanzmodell erfragt den Gesamtwärmestrom in diese Schicht als Ergebnisgröße ab: refType = `ConstructionBalanceModel`, id=<id der ConstructionInstance>, Name = `ActiveLayerHeatLoad`. Die Ergebnisgröße wird vom jeweils zuständigen Netzwerk bereitgestellt, wobei dieses "so tut", als wäre es ein `ConstructionBalanceModel`.

Berechnungsanpassung:

- Hydraulisches Rohrmodell: beim Beginn der Berechnung wird der Massestrom (angegeben für alle Rohre) durch den Skalierungsfaktor geteilt -> ergibt Massestrom durch Einzelrohr -> damit Strömungsgeschwindigkeit ... Druckverlust berechnet

- Thermische Modell: Wärmeübertragung berechnet für Einzelmodell -> bei Übergabe an die Fußbodenschicht Skalierung/Multiplikation mit Anzahl der Rohren

- Thermisches Modell: Energiebilanz -> Wärmestrom rein -> Skalierung über Massestrom wie bei hydraulischen Modell -> Auslasstemperatur vom Einzelmodell

**Comment by ghorwin** [comment]
Umsetzungsanforderungen:

Ausgabegrößen (zur automatisierten Kommunikation zwischen Modellen) des Rohrmodells sind:
- `m_fluidVolume` --> Fluidvolumen aller Rohre
- `m_massFlux` --> Massestrom durch alle Rohre
- `m_heatLoss` --> Wärmeverlust durch alle Rohre
- `y` -> Energiemenge aller Rohre
- `ydot`-> Änderungsrate aller Rohre
- `m_volumeFlow` -> Volumenstrom durch alle Rohre
